Kickstarter-iOS/Views/Controllers/FindFriendsViewController.swift

```diff
@@ -33,9 +34,32 @@ internal final class FindFriendsViewController: UITableViewController {
     self.navigationController?.setNavigationBarHidden(false, animated: animated)
   }
 
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableControllerStyle()
+      |> UIViewController.lens.title %~ { _ in Strings.Follow_friends() }
+
+    _ = self.navigationController?.navigationBar
+      ?|> baseNavigationBarStyle
+
+    _ = self.loadingIndicatorView
+      |> UIActivityIndicatorView.lens.activityIndicatorViewStyle .~ .white
+      |> UIActivityIndicatorView.lens.color .~ .ksr_navy_900
+  }
+
```

Comment:
just wondering why you moved `bindStyles` above `bindViewModel`? `bindStyles` is called after `bindViewModel` so we've been tending to show them in that order (vs alphabetical). At least that's how I prefer it. Not sure if there is a general consensus on this.